hoomd/snapshot.py

```diff
@@ -310,6 +310,10 @@ def replicate(self, nx, ny, nz):
         """
         self._cpp_obj.replicate(nx, ny, nz)
 
+    def wrap(self):
```

Comment:
In my opinion calling this function `wrap` is clear. However, you can also name it `wrap_to_box` which makes it a bit more specific.

Reply:
We use `wrap` in freud, and I think it's clear enough.